features/old/menu/dashboard.feature

```diff
@@ -16,9 +16,6 @@ Feature: Dashboard
     And I should see the link "0 DRAFTS" in the audience dashboard widget
     And I should see the link "0 MESSAGES" in the audience dashboard widget
 
-  Scenario: first API widget
-    And I should see "API" in the first api dashboard widget
```

Comment:
How could this work before if the old widget doesn't exist?

Reply:
This happens when tests are 💩. This scenario is asserting that "API" is on the screen inside the APIs widget:
<img width="1117" alt="Screenshot 2023-08-24 at 14 33 22" src="https://github.com/3scale/porta/assets/11672286/715c40a2-ab76-4ec8-8f93-57e6e02281c8">

Instead of removing this in the past, the selector _the first api dashboard widget_ was updated so that it target the 2 widgets Products and Backends...